external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -397,28 +422,33 @@ def save_jacobians(std_jacobians, dir_, filename="jacobians.npz"):
         np.savez(os.path.join(tmpdir, filename), **dumpable)
 
 
-@register_training_function("transformed", TransformedParameters)
-def train_function(
-    hyperparameters: TransformedParameters,
-    train_batches: tf.data.Dataset,
-    validation_batches: Optional[tf.data.Dataset],
-) -> PureKerasDictPredictor:
+def unbatch_datasets(train, test, filters):
```

Comment:
Is there a way to capture that this also applies the filters to the data in the function name? If not, can you add type hints and a docstring?